src/components/api.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+const api = axios.create({
+    baseURL: "https://nc-news-project-58d8.onrender.com/api/"
+})
+
+export function getAllArticles(paramsObj){
+    return api.get("/articles", {
+        params: paramsObj
+    })
```

Comment:
It would be good to get the articles off the correct keys here to take a bit of code out of your component
Great that you've separated out this into a separate api file, and love your use of axios' params and baseURL functionality.